kodiswift/mockxbmc/xbmcgui.py

```diff
@@ -62,3 +62,6 @@ def setThumbnailImage(self, thumb):
 
     def setArt(self, values):
         self.art = values
+
+class WindowXMLDialog:
```

Comment:
This should be improved, but it's not minor mock thus it was mocked in such way, to prevent runtime run...